wpi.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+
+import os
+
+sep = os.sep
+# If directory separator is backslash, escape it for regexes
+if sep == "\\":
+    sep += "\\"
+
+# Files and directories which should be included in or excluded from the update
+def folderExclude():
+    return "\.git" + sep + "|\.gradle" + sep + "|FRC_FPGA_ChipObject" + sep + \
```

Comment:
Might be more readable if this was done as `"|".join(["\.git" + sep, "\.gradle" + sep, ...])` with one line per location.